lib/form1010_ezr/service.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'hca/enrollment_system'
+require 'hca/configuration'
+require 'hca/ezr_postfill'
+
+module Form1010Ezr
+  class Service < Common::Client::Base
+    include Common::Client::Concerns::Monitoring
+    include VA1010Forms::ServiceUtils
+
+    STATSD_KEY_PREFIX = 'api.1010ezr'
+
+    # Due to using the same endpoint as the 10-10EZ (HealthCareApplication), we
+    # can utilize the same client configuration
+    configuration HCA::Configuration
+
+    FORM_ID = '10-10EZR'
+
+    # @param [Object] user
+    def initialize(user)
+      super()
+      @user = user
+    end
+
+    # @param [HashWithIndifferentAccess] parsed_form JSON form data
+    def submit_form(parsed_form)
+      begin
+        post_fill_required_fields(parsed_form)
+        validate_form(parsed_form)
+
+        formatted = HCA::EnrollmentSystem.veteran_to_save_submit_form(parsed_form, @user)
+        content = Gyoku.xml(formatted)
+        submission = soap.build_request(:save_submit_form, message: content)
+
+        response = perform(:post, '', submission.body)
```

Comment:
need `with_monitoring` block

Reply:
I thought the `with_monitoring` was going to be a separate issue. Isn't this the story for it? https://app.zenhub.com/workspaces/10-10-health-apps-5fff0cfd1462b6000e320fc7/issues/gh/department-of-veterans-affairs/va.gov-team/67787

Reply:
ok we can do it later

Reply:
KK, sounds good!